### DISCUSSION TITLE
可以支持中文版吗？

### DISCUSSION THREAD

**Discussion by fx607jv** [4IceG/luci-app-sms-tool-js#13] [General]
求个中文版本  中国地区

**Comment by 4IceG** [comment]
Hi @h88kmax ,
I saw that someone has already added the translation [zh_Hans](https://github.com/lazyoop/luci-app-sms-tool-js/blob/main/luci-app-sms-tool-js/po/zh_Hans/sms-tool-js.po), it's just a matter of compiling the language pack.

I can add it and compile it with the next package/repository update.

**Comment by fx607jv** [comment]
Thank you

**Comment by fx607jv** [comment]
测试接收短信验证码，第4条正常，超过5条就不显示了！接收的都是中文短信
![Uploading IMG_0096.png…]()

**Reply by 4IceG** [reply]
I don't see the image, so it's hard for me to answer. Sms-tool does not support all possible SMS types. I don't know if problem is with my LuCI package or sms-tool.

**Comment by 4IceG** [comment]
Hi @h88kmax ,
I compiled language pack [luci-i18n-sms-tool-js-zh-cn](https://github.com/4IceG/luci-app-sms-tool-js/releases/download/2.0.20-20240121/luci-i18n-sms-tool-js-zh-cn_240121.48551_all.ipk)

Check if you have the current version of sms-tool because there was an update that fixed the display of some messages.

**Reply by fx607jv** [reply]
我这里可以找到的sms-tool版本还是2022-03-21的X86版本最新的找不到！如果你有时间可以发一份最新的给我吗！谢谢 我的邮箱80962660@qq.com

**Reply by 4IceG** [reply]
@h88kmax Check my repository for the package https://github.com/4IceG/luci-app-sms-tool. I updated Makefile and the installation instructions for the sms-tool package.

**Reply by fx607jv** [reply]
非常感谢🙏我找到了最新版本sms-tool 短信卡死的问题解决了，谢谢大佬！要是可以支持发送中文短信就完美了！目前接收一切正常，就是发送中文短信对方显示全是？？？？？？

**Reply by 4IceG** [reply]
You need to test it yourself. I haven't sent messages in other languages, I'm curious myself :flushed:.

**Reply by fx607jv** [reply]
我已经测试了不行发送中文短信对方接收到的全是乱码！

**Reply by 4IceG** [reply]
Write here https://github.com/obsy/sms_tool/issues, maybe Cezary will be able to fix this sending problem.
